dev/standalone/integration-test-dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python
+
+WORKDIR /app/
+
+# Install test requirements first to make running the tests faster
+SHELL ["/bin/bash", "-c"] 
+COPY integration_requirements.txt /app/integration_requirements.txt
+RUN pip install virtualenv && \
+    virtualenv /tmp/gng_testing && \
+    source /tmp/gng_testing/bin/activate &&\
+    pip install -r integration_requirements.txt
```

Comment:
This can be simplified

1. `venv` is built-in in Python 3+
2. You can run `bin/python` directly without the need to activate the venv

```suggestion
RUN python -m venv /tmp/gng_testing && \
    /tmp/gng_testing/bin/python -m pip install -r integration_requirements.txt
```